### ISSUE TITLE
Mobs naturally spawning as invisible become impossible to fight

### ISSUE THREAD

**Issue by NighthowlerStudios** [capnkirok/betterinvisibility#13]
In some cases in vanilla, Spiders can spawn as invisible.  This effect can cause the eyes on the spider to completely disappear, making them near impossible to fight.

However, this carries over to modded scenarios, especially if dungeons are trying to spawn invisible entities.

I recommend adding a whitelist so that invisibility is not totally invisible if the id of the mob with the effect matches.

**Comment by capnkirok** [comment]
Very late reply. There's a config option to leave particle effects on. Is that not enough?

**Comment by NighthowlerStudios** [comment]
This issue a very long time ago was confirmed to be a client desync on singleplayer somehow.  What was happening is that a completely unrendered enemy was not taking any melee hits because the client's traces would constantly fail.

TL;DR, not your bug
